atlasdb-impl-shared/build.gradle

```diff
@@ -17,6 +17,7 @@ license {
 }
 
 dependencies {
+  api project(":timelock-api")
```

Comment:
Yeah. Understood where this comes from, and I think in general this is fine, but I also don't understand exactly how these subprojects are decomped. Specifically, at least in the past, internal products would run without timelock.

Should we duplicate TimestampLeaseName in atlasdb-api instead of doing this?

Reply:
No, it's too confusing internally. It's a pet peeve of mine to not have duplicate things flying around.

Reply:
I understand that, but libraries are hard